src/components/lib/localStorage.js

```diff
@@ -0,0 +1,19 @@
+export function loadLocally(key) {
+  try {
+    const serializedData = localStorage.getItem(key)
+    if (serializedData) {
+      return JSON.parse(serializedData)
+    }
+  } catch (error) {
+    console.log(error)
+  }
+}
+
+export function saveLocally(key, value) {
+  try {
+    const serializedData = JSON.stringify(value)
+    localStorage.setItem(key, serializedData)
```

Comment:
This way you can safe a line. I haven't implemented local Storage yet, so I'm curious: Is the async approach necessary? 
```suggestion
    localStorage.setItem(key, JSON.stringify(value))
```

Reply:
Good question, I haven't tried it without the async, but now I'll test it out to see if it's really necessary. Thanks!